src/backend/serial/scalar_mul/straus.rs

```diff
@@ -118,15 +118,14 @@ impl MultiscalarMul for Straus {
             .collect();
 
         // This puts the scalar digits into a heap-allocated Vec.
-        // To ensure that these are erased, pass ownership of the Vec into a
-        // ClearOnDrop wrapper.
-        let scalar_digits_vec: Vec<_> = scalars
+        // to be erased below with `Zeroize`.
+        let mut scalar_digits: Vec<_> = scalars
             .into_iter()
             .map(|s| s.borrow().to_radix_16())
             .collect();
-        let scalar_digits = ClearOnDrop::new(scalar_digits_vec);
```

Comment:
I wasn't able to use `Zeroizing` here because of the current `Zeroize` trait bounds for `Vec`, which are `Clone` + `Default` + `Zeroize`, and `Default` isn't impl'd for `[T: Default; N]` where `N > 32`, which is unfortunate.

Reply:
Update: fixed in `zeroize` 1.0